voluncheer/voluncheer/settings.py

```diff
@@ -91,8 +94,16 @@
 ]
 
 WSGI_APPLICATION = "voluncheer.wsgi.application"
+ASGI_APPLICATION = "voluncheer.asgi.application"
 
-
+CHANNEL_LAYERS = {
+    "default": {
+        "BACKEND": "channels_redis.core.RedisChannelLayer",
+        "CONFIG": {
+            "hosts": [("127.0.0.1", 6379)],
```

Comment:
What is the plan for AWS? Using a local container will not work especially when there is more than one instance running.

Reply:
I suggest looking into https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/ElastiCache-Getting-Started-Tutorials-Python.html for AWS integration.